lib/screens/registration/enter-phone.dart

```diff
@@ -0,0 +1,185 @@
+import 'dart:convert';
+
+import 'package:bc_app/constants/country_codes.dart';
+import 'package:flutter/material.dart';
+
+class EnterPhone extends StatefulWidget {
+  @override
+  State<StatefulWidget> createState() => _EnterPhone();
+}
+
+class _EnterPhone extends State<EnterPhone> {
+  static String dropdownValue = 'IN';
+  static String _phoneNumber;
+  static String isdCode = '+91';
+
+  @override
+  Widget build(BuildContext context) {
+    return SafeArea(
+      child: Scaffold(
+        appBar: AppBar(),
+        body: getBody(),
+        floatingActionButton: Padding(
+          padding: const EdgeInsets.only(right: 16),
+          child: FloatingActionButton(
+            child: new Icon(
+              Icons.navigate_next,
+              size: 48,
+            ),
+            backgroundColor: Color(0xff8FC7E8),
+          ),
+        ),
+      ),
+    );
+  }
+
+  getBody() {
```

Comment:
add the return type here.

```suggestion
Widget  getBody() {
```